integrations/server/test_covidcast_endpoints.py

```diff
@@ -106,38 +97,36 @@ class CovidcastEndpointTests(unittest.TestCase):
     def setUp(self):
         """Perform per-test setup."""
 
-        # connect to the `epidata` database and clear the `covidcast` table
-        cnx = mysql.connector.connect(user="user", password="pass", host="delphi_database_epidata", database="epidata")
+        # connect to the database and clear the tables
+        cnx = mysql.connector.connect(user="user", password="pass", host="delphi_database_epidata", database="covid")
         cur = cnx.cursor()
-        cur.execute("truncate table covidcast")
-        cur.execute('update covidcast_meta_cache set timestamp = 0, epidata = ""')
+
+        # clear all tables
+        cur.execute("truncate table signal_load")
+        cur.execute("truncate table signal_history")
```

Comment:
nitpick: quotation usage not consistent

Reply:
it bothers me too, but the inconsistency existed before this changeset so im gonna leave them for now